app/assets/javascripts/hamburger.js

```diff
@@ -0,0 +1,7 @@
+function openNav () {
+  document.querySelector("[data-behavior='myNav']").style.width = '100%';
```

Comment:
I would add the element type in front `[data-behavior='myNav']` just for explicitness